src/utils/rttc.ts

```diff
@@ -34,18 +34,13 @@ const typeOf = (v: Value) => {
   }
 }
 
-const isNumber = (v: Value) => typeOf(v) === 'number'
-const isString = (v: Value) => typeOf(v) === 'string'
-const isBool = (v: Value) => typeOf(v) === 'boolean'
-const isObject = (v: Value) => typeOf(v) === 'object'
-const isArray = (v: Value) => typeOf(v) === 'array'
+export const isNumber = (v: Value) => typeOf(v) === 'number'
```

Comment:
Reason for exporting all of these?

Reply:
Thought i was going to use them, but turns out I didn't